Mainframe3270/x3270.py

```diff
@@ -93,6 +111,22 @@ def open_connection(
         self.mf = Emulator(self.visible, self.timeout, extra_args)
         self.mf.connect(self.credential)
 
+    def _process_args(self, extra_args) -> List:
+        # this function should be the logically the same as append_args in ExecutableApp in py3270.py
```

Comment:
you can remove this comment since we should remove the whole `ExecutableApp` class from py3270.py